### PR TITLE
Implement rotate wrapper

### PR DESCRIPTION
**Pull request by Slaviiiii**

### CODE REVIEW

File: lib/jscad-fns/rotate.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export type RotateProps = {
+  angles: [number, number, number]
```
Comment (seveibar) [inline lib/jscad-fns/rotate.tsx:2]: see Translate for more details

```suggestion
  angles: Point3
```

**Review by seveibar — CHANGES_REQUESTED** [review]

File: lib/jscad-fns/rotate.tsx (hunk 1 above)
Comment (Slaviiiii) [reply lib/jscad-fns/rotate.tsx:2]: I see that Translate has args: [number, number, number]. 
How should I fix the issue there?

File: lib/create-host-config.ts [hunk 2]
```diff
@@ -240,11 +241,17 @@ export function createHostConfig(jscad: JSCADModule) {
       }
 
       case "translate": {
-        const { args, children } = props as JSX.IntrinsicElements["translate"]
+        const { args, children } = props as TranslateProps
```
Comment (seveibar) [inline lib/create-host-config.ts:244]: why did you revert this?

File: lib/create-host-config.ts [hunk 3]
```diff
@@ -240,11 +241,17 @@ export function createHostConfig(jscad: JSCADModule) {
...
         const childGeometry = renderChildren(children)
         return jscad.transforms.translate(args, childGeometry)
       }
 
+      case "rotate": {
+        const { angles, children } = props as RotateProps
```
Comment (seveibar) [inline lib/create-host-config.ts:250]: use jsx intrinsic, the rotate props aren't the same

**Review by seveibar — CHANGES_REQUESTED** [review]
see comments, i would just generally review the diff to make sure you didn't unnecessarily change anything

**Pull request closed without merging**